test/image/mocks/box_log_scale.json

```diff
@@ -0,0 +1,94 @@
+{
```

Comment:
Thanks very much for taking a shot a `violin` traces on log axes!

There seem to be a small problem with the position of the violin "box":

![image](https://user-images.githubusercontent.com/6675409/67705770-00774700-f98e-11e9-8301-c671ef93c64c.png)

the tall line at the right of the box should be drawn where the shading from the one-side violin ends.

Reply:
Actually this has nothing to do with your PR, all vertical one-sided violins have their box line shifted in the wrong direction e.g.: 

https://codepen.io/etpinard/pen/RwwZZzj?editors=1010

----

No need to address this here. I'll make a PR in the next few minutes fixing that.